examples/simple_regional_beta/main.tf

```diff
@@ -47,6 +47,10 @@ module "gke" {
   enable_binary_authorization = var.enable_binary_authorization
   pod_security_policy_config  = var.pod_security_policy_config
   release_channel             = "REGULAR"
+
+  # Disable workload identity
+  identity_namespace = null
```

Comment:
Should we update test to check for `"null"` since we added guard rails for that?

Reply:
No need, since GKE will not accept a `"null"` as a valid value (ie. creation fails).